test/test_unit_conversion_inputs.py

```diff
@@ -0,0 +1,62 @@
+"""
+Tests for unit conversion of inputs and defines.
+For further unit conversion tests see test_graphstate.py
+"""
+import pytest
+
+import fc
+
+
+@pytest.mark.xfail(strict=True, reason='Unit conversion not implemented yet.')
+def test_unit_conversion_inputs_initial_values():
+    # Test unit conversion for initial values of states and constants.
+
+    proto_file = 'test/protocols/test_unit_conversion_inputs_initial_values.txt'
+    proto = fc.Protocol(proto_file)
+    proto.set_output_folder('test_unit_conversion_inputs_initial_values')
+    proto.set_model('test/data/simple_ode.cellml')
+    proto.run()
+
+    # Check initial value of constant was set correctly
+    # The model has Na in mM with Na(0)=0mM, and dNa/dt=n, where n has initial value 1 mM/ms
+    # The protocol will change this to 3 M/ms, and then run for 5ms
+    Na = proto.output_env.look_up('cytosolic_sodium_concentration').array
+    assert len(Na) == 2
```

Comment:
I'd be inclined to write this as `assert Na = [0, 15000]` as then you get more informative output from pytest if it fails.